server/src/main/java/io/druid/segment/realtime/appenderator/Appenderators.java

```diff
@@ -124,8 +124,8 @@ public boolean isAnnounced(DataSegment segment)
         null,
         indexIO,
         indexMerger,
-        null,
-        null
+        MapCache.create(500000),
```

Comment:
@leventov thanks for the comments about the cache actually it is not really needed i will remove it.